src/Widgets/AgendaEventRow.vala

```diff
@@ -213,4 +361,13 @@ public class Maya.View.AgendaEventRow : Gtk.ListBoxRow {
             critical (e.message);
         }
     }
+
+    private void split_keywords (Category category) {
+        var words = category.get_builtin_keywords ().split (";", 20);
```

Comment:
Is there any specific reason to set a limit?
```suggestion
        var words = category.get_builtin_keywords ().split (";");
```

Reply:
No problem envisaged if this removed.